_validate/validate.py

```diff
@@ -30,22 +29,6 @@
 JsonObjT = typing.Dict[str, typing.Any]
 
 
-@enum.unique
-class ValidationErrorMessage(enum.Enum):
```

Comment:
I'm curious to know why this pattern was removed - it made the tests easier to read as well.

Reply:
I found the opposite, especially when the error messages changed meaning. It meant referring back to the original all the time. By putting the error message in the test it documents what the final output will be for each of the situations.